src/main/java/org/springframework/data/elasticsearch/core/query/FetchSourceFilterBuilder.java

```diff
@@ -15,19 +15,31 @@
  */
 package org.springframework.data.elasticsearch.core.query;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
```

Comment:
needed here?

Reply:
for parsing source filters with missing properties from `@Query` annotation 
e.g.
```
{
  "_source": { "excludes": ["field"] } 
}
```